boards/shields/x_nucleo_iks01a1/doc/x_nucleo_iks01a1.rst

```diff
@@ -0,0 +1,63 @@
+.. _x-nucleo-iks01a1:
+
+X-NUCLEO-IKS01A1: MEMS Inertial and Environmental Multi sensor shield
+#####################################################################
+
+Overview
+********
+The X-NUCLEO-IKS01A1 is a motion MEMS and environmental sensor
+evaluation board system, compatible with the Arduino UNO R3 connector
+layout.  It includes an STMicroelectronics’ LSM6DS0 3-axis accelerometer
+and 3-axis gyroscope, the LIS3MDL 3-axis magnetometer, the HTS221
+humidity and temperature sensor, and the LPS25HB pressure sensor.
+
+The X-NUCLEO-IKS01A1 interfaces with the STM32 microcontroller via the I2C pin,
+and it is possible to change the default I2C port.
```

Comment:
When I read this it sounds weird.

Perhaps it should say:

The X-NUCLE0-IKS01A1 board allows other boards to interface with the STM32 via an I2C interface.

Reply:
I like!  (The current line is a paste http://www.st.com/en/ecosystems/x-nucleo-iks01a1.html)